src/api/swagger/restapi/configure_monocular.go

```diff
@@ -37,10 +37,14 @@ func configureAPI(api *operations.MonocularAPI) http.Handler {
 	}
 	// configure the api here
 	chartsImplementation := cache.NewCachedCharts(config.Repos)
+	// Run foreground repository refresh
+	chartsImplementation.Refresh()
```

Comment:
Move refresh to foreground during app bootstrap because technically before we were blocking anyways but via a `Sync.Lock` which took into effect once called doPeriod(). This method was brittle and prone to race conditions.